MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+include README.md
+recursive-include cmsplugin_feed_ai *.*
```

Comment:
I'm not sure how it behaves in the sdist package. If it gets unpacked into `site-packages`, it's not nice. Should really check that...